server/task/task-memory/src/test/java/org/apache/james/task/eventsourcing/TaskAggregateTest.java

```diff
@@ -85,6 +85,35 @@ void givenInProgressTaskEmitEventWhenUpdateAdditionalInformationCommand() {
             .containsExactly(AdditionalInformationUpdated.apply(ID, history.getNextEventId(), new MemoryReferenceWithCounterTask.AdditionalInformation(3, timestamp)));
     }
 
+    @Test
+    void givenInProgressTaskWithOneNewerUpdateEventEmitEventWhenUpdateAdditionalInformationCommand() {
```

Comment:
your test names are barely readable, mainly because they miss a "Should" keyword.